### PR TITLE
Update codeowners

### PR DESCRIPTION
**Pull request by lrapoport-cf**
Fixes N/A.

**What this PR solves / how to test:** https://github.com/cloudflare/workers-sdk/pull/4774 was meant to remove required review from package owners for `workers-sdk` Version Packages release PRs. However, it did not do so because `package.json` files still require package owner review. This PR removes that ownership.

**Author has addressed the following:**

- Tests
  - [ ] Included
  - [x] Not necessary because: this is only a change to code ownership and there aren't tests associated with this
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] Included
  - [x] Not necessary because: this is only a change to code ownership
- Associated docs
  - [ ] Issue(s)/PR(s):
  - [x] Not necessary because: this is only a change to code ownership

**Note for PR author:**

We want to celebrate and highlight awesome PR review! If you think this PR received a particularly high-caliber review, please assign it the label `highlight pr review` so future reviewers can take inspiration and learn from it.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -22,3 +22,4 @@
 # Only wrangler-admins are needed to approve releases
 # Product teams can request hold on release by reaching out to wrangler-admins
 **/CHANGELOG.md @cloudflare/wrangler-admins
+**/package.json @cloudflare/wrangler
```
Comment (Skye-31) [inline CODEOWNERS:25]: Should that be?
```suggestion
**/package.json @cloudflare/wrangler-admins
```

Comment (lrapoport-cf) [reply CODEOWNERS:25]: @Skye-31 i considered that but then `package.json` changes in non-release PRs would require wrangler-admin team approval. i think the default of wrangler team approval would address the intent here

Comment (Skye-31) [reply CODEOWNERS:25]: Good point!

Comment (lrapoport-cf) [reply CODEOWNERS:25]: 😀

**Pull request merged**